static/app/views/performance/browser/webVitals/utils/queries/useTransactionWebVitalsQuery.tsx

```diff
@@ -9,6 +9,7 @@ type Props = {
   enabled?: boolean;
   limit?: number;
   opportunityWebVital?: WebVitals | 'total';
+  query?: string;
```

Comment:
No need to address this, just a comment,

A pattern i've been using for resource module is making this
`queryConditions: string[]`, and you pass in each condition as an element, join them with a space at the end.
There's pros and cons to each, but I think typically this is easier to read/cleaner.

Reply:
Yeah I like this too, I'll keep this update for a future pr, don't want to overload this one too much